src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -178,19 +213,6 @@ const DomainFragment = ({
     if (searchText.length > 0) fetchData();
   };
 
-  const renderRecentSearch = (index) => {
-    return (
-      index === 0 &&
-      !withoutRecent && (
-        <RecentSearch
-          shown={isFirstTimeOpen}
-          setSearchText={setSearchText}
-          setIsFirstTimeOpen={setIsFirstTimeOpen}
-        />
-      )
-    );
-  };
-
   const renderDefaultImage = () => {
     return (
       <MemoDomainProfilePictureEmptyState
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `renderDefaultImage` returns a default image when no image is provided. This is a good practice for handling missing data.
```javascript
const renderDefaultImage = () => {
  return (
    <MemoDomainProfilePictureEmptyState
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->